test/src/test/java/org/corfudb/runtime/view/ManagementViewTest.java

```diff
@@ -49,6 +50,13 @@ public void invokeFailureHandler()
                 .build();
         bootstrapAllServers(l);
 
+        // Reduce test execution time from 15+ seconds to about 8 seconds:
+        // Set aggressive timeouts for surviving MS that polls the dead MS.
+        ManagementServer ms = getManagementServer(9001);
+        ms.getCorfuRuntime().getRouter("test:9000").setTimeoutConnect(50L);
```

Comment:
If you have a recommendation for selecting a different constant in the PARAMETERS object, I'd appreciate it.  Otherwise, IMO the test is should be removed & refactored.